### PR TITLE
add color grading with adjust.py

### PR DESCRIPTION
**Pull request by SnowzNZ**
same functionality as [blur](https://github.com/f0e/blur/blob/29937d4f733b3ec670a615295488e481ae174cfe/blur/script_handler.cpp#L126)

### CODE REVIEW

**Comment by couleurm** [comment]
the problem with this is that, let's say your recipe is configured like so:
```ini
[color grading]
saturation: 1.0
brightness: 1.0
contrast: 1.1
```
it won't apply because some of them are at 1.0

**Comment by SnowzNZ** [comment]
> the problem with this is that, let's say your recipe is configured like so:
> 
> ```ini
> [color grading]
> saturation: 1.0
> brightness: 1.0
> contrast: 1.1
> ```
> 
> it won't apply because some of them are at 1.0

It will apply:
https://user-images.githubusercontent.com/98007575/227745789-a1a33a6e-19f3-4d4d-8988-ca22e817419d.mp4

File: target/jamba.vpy [hunk 1]
```diff
@@ -224,4 +224,21 @@ if (flb := rc['flowblur'])['enabled'].lower() in YES and rc['flowblur']['enabled
         verb(clip)
 # flowblur
 
+cg = rc["color grading"]
+
+brightness = float(cg["brightness"])
+contrast = float(cg["contrast"])
+saturation = float(cg["saturation"])
+hue = float(cg["hue"])
```
Comment (SnowzNZ) [inline target/jamba.vpy:232]: Assumes hue is set in recipe.ini

Comment (couleurm) [reply target/jamba.vpy:232]: it will be present in defaults.ini either way 👍

**Pull request closed without merging**